### DISCUSSION TITLE
Pre DS3/4

### DISCUSSION THREAD

**Discussion by israpps** [israpps/wLaunchELF_ISR#14] [Q&A]
The last commit before implementation of PlayStation 3 and 4 dualshocks

That new feature added a bug wich causes the program to atpmatically return into devices root when accessing `mc0:/`

<hr /><em>This discussion was created from the release <a href='https://github.com/israpps/wLaunchELF_ISR/releases/tag/'>Pre DS3/4</a>.</em>

**Comment by Frander99** [comment]
Does this version support exFAT ?

**Reply by israpps** [reply]
It will never support exfat, only latest uLaunchELF will do

**Reply by israpps** [reply]
For that, latest toolchain is needed, to get there, stability must be sacrificed

**Reply by Frander99** [reply]
Pero piensas sacar una version que soporte exFAT ? uso tu version por el peso y que muestra la fecha de los arvhivos, cosa que no sucede en el wLaunchelf...

**Reply by Frander99** [reply]
Y no utilizo formato exFat aun porque hay muchas otras aplicaciones que aun no son compatibles... Por ejemplo CheatDevice

**Reply by israpps** [reply]
> Pero piensas sacar una version que soporte exFAT ? uso tu version por el peso y que muestra la fecha de los arvhivos, cosa que no sucede en el wLaunchelf...

 El problema de la fecha de los archivos fue culpa de la implementación de los driver exfat, me parece que ya se corrigió/o al menos está por corregír

**Reply by Frander99** [reply]
Ok, osea que en tu version no utilizas esos drivers... Bueno esperare a ver.

**Reply by israpps** [reply]
> Ok, osea que en tu version no utilizas esos drivers... Bueno esperare a ver.

Este mod de uLaunchELF está basado en una versión muy estable del 2019, y que utiliza un SDK de la misma época...

Esto deja mi mod atrasado respecto a las funcionalidades más recientes del sdk...

Mucho no me importa ya que el propósito de esta versión era ofrecerle al usuario la máxima estabilidad posible en las funciones generales del programa, sobre todo en administración de disco duro interno y memory cards

**Reply by Frander99** [reply]
SI, eso es lo mas importante para mi...
